lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lerna": "2.0.0-rc.4",
+  "lerna": "2.9.0",
```

Comment:
@bjankord @mjhenkes We specify lerna to be v `^2.1.2` in core, however out lerna.json specifies `2.1.2`. Do we need to bump ours or lock to the lerna version?

Reply:
We should take a look at upgrading.